perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -818,6 +840,7 @@ def Cleanup(self, vm):
 MYSQL_SERVICE_BENCHMARK_DICTIONARY = {
     providers.GCP: GoogleCloudSQLBenchmark(),
     providers.AWS: RDSMySQLBenchmark()}
+DATA_LOAD_RESULTS = []
```

Comment:
1. Add a comment explaining why this variable needs to be global.

2. perhaps rename it to "DATA_LOADING_RESULTS"? Similarly for all phrases below that says "data load results", consider renaming them to "data loading results".